### PR TITLE
Add API conversion interface for MaxPooling1D layer

### PR DESCRIPTION
**Pull request by jihobak**
Add API conversion interface for MaxPooling1D layer.

`if args:`
`# The first entry in `args` is `self`. So, args[1:]`
`signature += ', '.join([str(arg) for arg in args[1:]])`

I add above to show a warning messgewhen people use just one old argument and one without argument like `old_layer = keras.layers.MaxPool1D(2, border_mode='valid', name='maxpool1d')`

I would like ask some advice.

When people use only old version of keyword argument, like `old_layer = keras.layers.MaxPool1D(pool_length=2, border_mode='valid', name='maxpool1d')`

warning message will be ``Update your `MaxPooling1D` layer call to the Keras 2 API: `MaxPooling1D(, pool_size=2, padding="valid")``. 

If I try to get rid of the first comma, code is getting messy. Is there any advice or style using in this case on Keras?

### CODE REVIEW

File: keras/legacy/interfaces.py [hunk 1]
```diff
@@ -74,3 +74,50 @@ def wrapper(*args, **kwargs):
...
+                    raise_duplicate_arg_error(old_arg, new_arg)
+                arg_value = kwargs.pop(old_arg)
+                kwargs[new_arg] = arg_value
+                converted.append((new_arg, arg_value))
+
+        if converted:
+            signature = '`MaxPooling1D('
+            if args:
+                # The first entry in `args` is `self`. So, args[1:]
+                signature += ', '.join([str(arg) for arg in args[1:]])
```
Comment (fchollet) [inline keras/legacy/interfaces.py]: In case `padding` is a positional argument, quote characters will be required around it here (since it's a string). E.g. don't print `MaxPooling1D(3, valid)`, instead print `MaxPooling1D(3, "valid")`.

File: tests/keras/legacy/interface_test.py [hunk 2]
```diff
@@ -28,5 +28,12 @@ def test_dense_legacy_interface():
     assert json.dumps(old_layer.get_config()) == json.dumps(new_layer.get_config())
 
 
+@keras_test
+def test_maxpool1d_legacy_interface():
+    old_layer = keras.layers.MaxPool1D(pool_length=2, border_mode='valid', name='maxpool1d')
```
Comment (fchollet) [inline tests/keras/legacy/interface_test.py]: Please also add a test checking the use of 1 or 2 positional arguments.

File: keras/legacy/interfaces.py (hunk 1 above)
Comment (fchollet) [inline keras/legacy/interfaces.py]: Actually, you can just use the `convert_legacy_kwargs` utility function I just added.

Comment (jihobak) [inline keras/legacy/interfaces.py]: @fchollet 
Thanks, I get what you pointed out. 
but isn't it impossible to test `MaxPooling1D(3, 'valid')`?
padding is can't be positional argument without strides value because there is 'stride' argument

So, I think there will not be the case that you pointed out

I looking forward to your reply, 

Thanks

Comment (jihobak) [inline keras/legacy/interfaces.py]: @fchollet  by the way, Where did you put 'convert_legacy_kwargs'?

Comment (fchollet) [inline keras/legacy/interfaces.py]: Right here: https://github.com/fchollet/keras/blob/keras-2/keras/legacy/interfaces.py#L15

Comment (jihobak) [inline keras/legacy/interfaces.py]: @fchollet I appreciate it ,
what about MaxPooling1D(3, 'valid') case? I think it it impossible.

Comment (fchollet) [inline keras/legacy/interfaces.py]: You're right. In that case we should make sure the user does not pass more than 1 positional argument.

Comment (jihobak) [inline keras/legacy/interfaces.py]: @fchollet 
What about making sure the user does not pass positional argument? because Maxpool1d doesn't have any positional argument.

`def __init__(self, pool_size=2, strides=None,
                 padding='valid', **kwargs)`

Comment (fchollet) [inline keras/legacy/interfaces.py]: `pool_size` would be commonly used as positional argument.

**Comment by jihobak** [comment]
I squash and make a new pull request
